src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -149,6 +149,7 @@ private static function parseDsn($dsn)
     {
         $engine = substr($dsn, 0, strpos($dsn, ':'));
         $tags = ['db.engine' => $engine];
+        $tags[Tag::DB_SYSTEM] = $tags['db.engine'];
```

Comment:
The driver name is not always the same as the standardized value for that system. For instance, `pgsql` is the driver name, but the standardized name for that is `postgresql`. You need to do a mapping, at least for this one case (maybe others), hence my previous comment about going through the docs.

Reply:
Fixed this.